tests/enterprise-contract/contract.go

```diff
@@ -23,7 +23,7 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
-var _ = framework.EnterpriseContractSuiteDescribe("Enterprise Contract E2E tests", Label("ec"), func() {
+var _ = framework.EnterpriseContractSuiteDescribe("Conforma E2E tests", Label("cli"), func() {
```

Comment:
This might possibly have some side-effects, e.g. if the label is references anywhere. Could consider leaving the label as "ec" to avoid that. (Disregard if you considered that already and figured out where/how the label is used.)